bHY6yh-d34855/cod-pol7d7_bHY6yh-d34855.xml

```diff
@@ -52,32 +52,28 @@
    <text xml:lang="la">
       <front>
          <div xml:id="starts-on">
-           <pb ed="#K" n="46-r"/>
+           <pb ed="#K" n="78-v"/>
            <cb ed="#K" n="a"/>
          </div>
       </front>
       <body>
          <div xml:id="bHY6yh-d34855"><!-- p2l4-r12 -->
             <head xml:id="bHY6yh-d34855-Hd1e108">Rubrica 12</head>
-            <head xml:id="bHY6yh-d34855-Hd1e111" type="question-title">De satisfactione illate iniurie</head>
+            <head xml:id="bHY6yh-d34855-Hd1e111" type="question-title">De pace seu concordia</head>
```

Comment:
@Kat, if these are the same rubric, the "head" should be left alone. The `head@question-title` is metadata for the section regardless of manuscript. 

The "rubric" or chapter heading in the specific manuscript should be transcribed in the first paragraph below  which you have done

But are you sure these are aligned properly. The venice text you copied over says rubric and you've changed this to rubric 13 and the topic seems quite different